lib/warder/version.rb

```diff
@@ -1,4 +1,4 @@
-# define warder version
+# defines warder version
 module Warder
-  VERSION = '0.2.3'
+  VERSION = '0.2.4'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.